### PR TITLE
Clip cover_multistep_options's option policies with limits of action space. 

### PR DESCRIPTION
**Pull request by ashayathalye**
This allows for flexibility in defining variable step policies in cover_multistep_options because the Pick() and Place() ground truth option policies now clip the low-level-action by the bounds of the action space. If we want policies that take more steps to go from point to point, we can reduce the limits of the action space.

### CODE REVIEW

File: src/settings.py [hunk 1]
```diff
@@ -18,6 +18,9 @@ class GlobalSettings:
     cover_block_widths = [0.1, 0.07]
     cover_target_widths = [0.05, 0.03]
 
+    # cover_multistep_options parameters
+    action_space_limits = [-np.inf, np.inf]
```
Comment (tomsilver) [inline src/settings.py:22]: can you change this to `cover_multistep_options_action_space_limits`? or if that's too long, `cover_multistep_action_limits`

**Review by tomsilver — COMMENTED** [review]
one small comment, otherwise lgtm thanks!

**Comment by ronuchit** [comment]
LGTM after tom's suggestion

File: src/settings.py (hunk 1 above)
Comment (ashayathalye) [reply src/settings.py]: Will do the latter.

**Pull request merged**